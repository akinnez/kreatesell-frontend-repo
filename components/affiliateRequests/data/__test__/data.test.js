import { requestsData } from "..";

it("renders a snapshot of the mock data requestsData", () => {
  expect(requestsData).toMatchSnapshot();
});
