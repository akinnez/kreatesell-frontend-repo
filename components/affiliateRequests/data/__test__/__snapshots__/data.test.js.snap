// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of the mock data requestsData 1`] = `
Array [
  Object {
    "commission": 18,
    "currency": "NGN",
    "date_created": "10/11/2021 8:25:21 PM",
    "id": 1,
    "kreator_name": "Jonah Jameson",
    "performance": Object {
      "sold": 50,
      "visit": 123,
    },
    "product_name": "UI Design Introduction",
    "product_type": 2,
    "sales_price": "4568",
    "status": "Approved",
  },
  Object {
    "commission": 15,
    "currency": "NGN",
    "date_created": "01/04/2021 8:25:21 PM",
    "id": 2,
    "kreator_name": "Mary Parker",
    "performance": Object {
      "sold": 10,
      "visit": 22,
    },
    "product_name": "Web Development Course",
    "product_type": 3,
    "sales_price": "5000",
    "status": "Pending",
  },
  Object {
    "commission": 30,
    "currency": "NGN",
    "date_created": "08/13/2021 8:25:21 PM",
    "id": 3,
    "kreator_name": "Peter Parker",
    "performance": Object {
      "sold": 20,
      "visit": 25,
    },
    "product_name": "Cooking Recipe",
    "product_type": 1,
    "sales_price": "10000",
    "status": "Declined",
  },
  Object {
    "commission": 5,
    "currency": "NGN",
    "date_created": "02/14/2021 8:25:21 PM",
    "id": 4,
    "kreator_name": "Lil Wayne",
    "performance": Object {
      "sold": 898,
      "visit": 1000,
    },
    "product_name": "My Music",
    "product_type": 1,
    "sales_price": "2500",
    "status": "Pending",
  },
]
`;
